<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>829</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>579</width>
    <height>829</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>579</width>
    <height>829</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>551</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="dataP">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Dzisiaj jest &quot;dzień obecny&quot;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>541</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Jakość powietrz na dzień dzisiejszy</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>531</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/wska/wska.PNG);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../wska.PNG</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>400</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Obecna stacja pomiarowa z okolocy: &quot;stacja&quot;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>440</y>
        <width>411</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/wyk/wyk.PNG);
background-image: url(:/wyk/wyk.PNG);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../wyk.PNG</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mapP">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>411</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mapa jakości powietrza</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>150</y>
        <width>551</width>
        <height>511</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/nowyPrzedrostek/C:/Users/Wojtek/Desktop/UCZELNIA/Semestr 6/komunikacja człowiek-komputer/projekt/mapa.jpg);
image: url(:/mapa/mapa.jpg);
background-image: url(:/nowyPrzedrostek/C:/Users/Wojtek/Desktop/UCZELNIA/Semestr 6/komunikacja człowiek-komputer/projekt/mapa.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>:/nowyPrzedrostek/C:/Users/Wojtek/Desktop/UCZELNIA/Semestr 6/komunikacja człowiek-komputer/projekt/mapa.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="optionP">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>260</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>strona w pracy</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="mapBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>760</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mapa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dataBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>760</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stacje/Dane</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>760</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Opcje</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="obraz1.qrc"/>
 </resources>
 <connections/>
</ui>
